lib/ffmpeg/util.py

```diff
@@ -85,6 +85,11 @@ def have_ffmpeg_encoder(encoder):
   result = try_call(f"{exe2os('ffmpeg')} -hide_banner -encoders | awk '{{print $2}}' | grep -w {encoder}")
   return result, encoder
 
+@memoize
+def have_string_api(encoder):
```

Comment:
How about make this more generic?  Call it `have_ffmpeg_encoder_options`, similar to `have_ffmpeg_filter_options`.